sockeye/translate.py

```diff
@@ -108,7 +118,8 @@ def run_translate(args: argparse.Namespace):
                                           store_beam=store_beam,
                                           strip_unknown_words=args.strip_unknown_words,
                                           skip_topk=args.skip_topk,
-                                          sample=args.sample)
+                                          sample=args.sample,
+                                          fixed_length_ratio=constant_length_ratio)
```

Comment:
maybe rename ether the variable to fixed_length_ratio or the argument to constant_length_ratio

Reply:
done